config/routes.rb

```diff
@@ -5,4 +5,5 @@
   resources :wizard, only: [:create]
   resources :orders, only: [:create]
   resources :payments, only: [:create]
+  get :shipping_rates, to: "shipping_rates#simulate_shipping_rates"
```

Comment:
Only to keep in line with what those actions represent - I didn't feel like the method we needed fit index, show, create, or edit - since it's not really doing any of those actions. I thought a custom action was warranted here since it's performing a nonstandard action.